docs/en/status-wallet/add-watch-only-accounts.md

```diff
@@ -8,9 +8,32 @@ hide:
 
 # Add watch-only accounts
 
- :octicons-tools-24: In progress
+You can add any Ethereum address as a watch-only account, including addresses that don't belong to you. Once you add them, you can keep track of their balance and transactions in your Status Wallet app.
```

Comment:
Your intro can improve a bit to clarify (1) what's a watch-only account and (2) why it can be helpful. Your intro paragraph touches on this only superficially. 

I would include something like this before your paragraph:

> You can add watch-only accounts to your Status Wallet to monitor other crypto wallet balances and activity.

... and something like this after your paragraph:

> This can be helpful to view your balances in different wallets or to monitor balances of Ethereum addresses you're interested in.

Feel free can adjust these sentences to avoid repeating information.

Reply:
✅ Done! I slightly modified your suggestions to go into a bit more detail.